MaxText/inference_microbenchmark.py

```diff
@@ -27,10 +28,29 @@
 import maxtext_utils
 import pyconfig
 
-
 _WARMUP_ITERS = 2
 
 
+def debug_kv_cache(kv_cache):
+  singler_kv_cache = kv_cache["cache"]["decoder"]["layers_0"]["self_attention"]["AttentionOp_0"]
+  for cache_key in singler_kv_cache.keys():
+    cache_element = singler_kv_cache[cache_key]
+    print(f"{cache_key}:")
+    if type(cache_element) == flax.linen.spmd.LogicallyPartitioned:
+      cache_element = cache_element.value
+    jax.debug.print("  shape: {shape}", shape=cache_element.shape)
```

Comment:
Nit: This is a dense series of lines, some whitespace would help make it more readable. 

A small thing that you can take or leave related to density is that in `jax.debug.print()` you can ignore the var naming if you are only printing one var. Like this `jax.debug.print(" sharding: {}", cache_element.sharding)`.

Reply:
Done